design-proposals/eim-platform-bundle-improvements.md

```diff
@@ -0,0 +1,130 @@
+# Design Proposal: Improvements on Platform Bundle integration with EIM
+
+Author(s): Tomasz Osiński
+
+Last updated: 14.05.2025
+
+## Abstract
+
+Platform Bundle is defined as a collection of OS-level files, installation scripts, configurations and packages.
+It's a concept to make any OS profile supported by EIM extensible and customizable. For example,
+we can have different flavors of Ubuntu/EMT images that support various use cases.
+It's a mechanism that allows EN support features beyond what EIM supports currently.
+
+Platform Bundle is already integrated with EIM and gives ability to provide custom cloud-init template
+and Installer script (for Ubuntu).
+For instance, the `Ubuntu with Intel extensions` OS profile uses Installer script that is curated
+by EEF (Edge Enablement Framework) to install additional GPU drivers.
+
+However, the current Platform Bundle design has the following problems:
+
+1. Co-ownership of cloud-init template and Installer script between EIM and EEF,
```

Comment:
I think what you want to say here is that platform bundle completely overrides the internal one, effectively leaving 0 room for EIM curated scripts that we know work